tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP</categories>
+          <categories>math-ph math.MP astro-ph astro-ph.IM</categories>
```

Comment:
I don't think that using a specially crafted record is 'cheating', actually, I think that it's the way to go, specially for acceptance/functional tests, where adding more records and test runs is a lot more costly. I don't see how this adds false security either, the test is really testing a specification, not a tautology.

I would agree though that it might be more elegant to move the set of 'experimental categories' to a configuration value, and tweak that on the test to make the record experimental instead.